erizo/src/erizo/NicerConnection.h

```diff
@@ -105,6 +105,8 @@ class NicerConnection : public IceConnection, public std::enable_shared_from_thi
   nr_ice_handler* ice_handler_;
   std::promise<void> close_promise_;
   std::promise<void> start_promise_;
+  boost::mutex close_mutex_;
+  boost::mutex closeSync_mutex_;
```

Comment:
please consider using snake_case here: `close_sync_mutex_`

Reply:
@Trisfald The only thing we need to merge this PR is this change, could you apply it please? thanks a lot!

Reply:
Done! By bad, I don't know how I missed your prior comment